src/places/types/placeRange.ts

```diff
@@ -0,0 +1 @@
+export type PlaceRange = { topLeft: { lat: number; long: number }; bottomRight: { lat: number; long: number } } | null;
```

Comment:
1. Please avoid creating types like that: `type X = 'something' | null`. Instead, handle `null` while using the type. So for instance, in this particular case: `function getPlaceRange(): PlaceRange | null`.
2. Code formatting should be fixed - never use one-line formatting for complex types
3. Also, for more complex types, and for avoiding duplication, this could be splitted like this:
```typescript
export type PlaceRange = {
  topLeft: Coordinates;
  bottomRight: Coordinates;
};

type Coordinates = {
  lat: number;
  long: number;
};
```